src/lib/operators/join_hash/join_hash_steps.hpp

```diff
@@ -191,6 +191,16 @@ class PosHashTable {
     return _offset_hash_table.find(casted_value) != _offset_hash_table.end();
   }
 
+  // Return the size of the hash table and the number of positions stored (in case the mode is AllPositions). We use
+  // the unified position list that is created in finalize(). For semi/anti* joins the number of positions is 0.
```

Comment:
if you use the unified positions list, maybe assert that it was already created?